jazz-ui/src/app/pages/service-cost/service-cost.component.ts

```diff
@@ -193,6 +193,9 @@ export class ServiceCostComponent implements OnInit {
 	regSelected:string;
 	//  = this.regList[0];
 	 
+	refresh(){
+		// this.refreshCostData();
```

Comment:
remove?

Reply:
every component has a refresh function, We will be needing this in future when the component is enabled

Reply:
so should this be added in future then rather than now.